src/CommandHandler.cxx

```diff
@@ -353,6 +351,8 @@ void CommandHandler::handleStreamMasterStop(std::string const &Command) {
   if (!MasterPtr) {
     return;
   }
+  LOG(Sev::Critical, "{}", Command);
```

Comment:
Is there some other place where we can log (maybe in Debug mode only) the command? It could be helpful for debugging. If not, I would suggest to keep this log and set it to Sev::Debug.